spaceflights/{{ cookiecutter.repo_name }}/.gitignore

```diff
@@ -20,7 +20,7 @@ logs/**
 !.gitkeep
 
 # keep also the example dataset
-!data/01_raw/iris.csv
+!data/01_raw/*.csv
```

Comment:
Spaceflights also has an excel file right?

Reply:
Thanks. Updated. Also updated the import @lorenabalan